pipeline/outputs/oci-logging-analytics.md

```diff
@@ -0,0 +1,187 @@
+|:------------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:--------|
+| oci_config_in_record    | If set to true, the following oci_la_* will be read from the record itself instead of the output plugin configuration.                                                                        | false   |
+| oci_la_log_group_id     | The OCID of the Logging Analytics Log Group where the logs must be stored. This is a mandatory parameter                                                                                      |         |
+| oci_la_log_source_name  | The Logging Analytics Source that must be used to process the log records. This is a mandatory parameter                                                                                      |         |
+| oci_la_entity_id        | The OCID of the Logging Analytics Entity	                                                                                                                                                     |         |   
+| oci_la_entity_type	     | The entity type of the Logging Analytics Entity	                                                                                                                                              |         |
+| oci_la_log_path	        | Specify the original location of the log files	                                                                                                                                               |         |
+| oci_la_global_metadata	 | Use this parameter to specify additional global metadata along with original log content to Logging Analytics. The format is 'key_name value'. This option can be set multiple times          |         |
+| oci_la_metadata	        | Use this parameter to specify additional metadata for a log event along with original log content to Logging Analytics. The format is 'key_name value'. This option can be set multiple times |         |
+
+## TLS/SSL
+
+OCI Logging Analytics output plugin supports TTL/SSL, for more details about the properties available and general configuration, please refer to the [TLS/SSL](../../administration/transport-security.md) section.
+
+## Getting Started
+
+### Prerequisites
+
+- OCI Logging Analytics service must be onboarded with the minumum required policies, in the OCI region where you want to monitor. Refer [Logging Analytics Quick Start](https://docs.oracle.com/en-us/iaas/logging-analytics/doc/quick-start.html) for details.
+- Create OCI Logging Analytics LogGroup(s) if not done already. Refer [Create Log Group](https://docs.oracle.com/en-us/iaas/logging-analytics/doc/create-logging-analytics-resources.html#GUID-D1758CFB-861F-420D-B12F-34D1CC5E3E0E) for details.
```

Comment:
Is this link stable? Just a little concerned with the need for a GUID.

Reply:
This is a stable link @patrick-stephens